common/stages/Functions/library

```diff
@@ -342,3 +342,31 @@ get_partition_node()
 
     return 0
 }
+
+#-----------------------------------------------------------
+# Usage: version_ge <v1> <v2>
+# Parse and compare versions formatted as "major.minor.micro".
+# Return 0 if v1 >= v2, else return 1.
+version_ge() {
```

Comment:
True but I am not familiar with ash, does it have the ```[[``` built-in? The rest of the script pretty much imply the opposite (only use ```[```).